src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCause.java

```diff
@@ -659,7 +659,7 @@ public void setLastFailedBuildUrl() {
 
         @Override
         public String getDisplayName() {
-            return null;
+            return "";
```

Comment:
IIRC it can be already removed in this version... or not?

Reply:
Even on 1.625 it was `@Nonnull`